integration/setup/setup.go

```diff
@@ -35,17 +34,37 @@ const (
 // WrapTestMain setup and teardown e2e testing environment.
 func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
 	var r int
+	var l micrologger.Logger
```

Comment:
There is a logger already created. Check where the frameworks above come from. Lets just dispatch the logger from there to here.

Reply:
Where? I check all three structs and none of them has the micrologger property? Is it hidden somewhere?

I checked
```
g *framework.Guest
h *framework.Host
m *testing.M
```

Reply:
`WrapTestMain` is called with 3 arguments. Make 4 out of that.